Makefile

```diff
@@ -32,3 +32,9 @@ notice:
 
 build-docker-ci:
 	docker build -t crawler-ci .buildkite
+
+docker-build: $(DOCKERFILE_PATH)
+	docker build -f $(DOCKERFILE_PATH) -t $(DOCKER_IMAGE_NAME):$(VERSION)-SNAPSHOT .
```

Comment:
~doesn't `-SNAPSHOT` get added via the `USE_SNAPSHOT` stuff in `publish-common.sh`, so would be redundant here?~

actually maybe a better question is: why are we adding this Makefile target (and the one elow) if we do our image buliding and pushing via scripts in `.buildkite/scripts/publish`?

Reply:
I included this so we'd have a manual way to publish from local, but thinking more on it, I don't think we'll ever do that. It can be deleted.